src/client.ts

```diff
@@ -3198,10 +3198,10 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
      * @returns a promise which resolves when the keys have been imported
      */
     public importRoomKeys(keys: IMegolmSessionData[], opts?: IImportRoomKeysOpts): Promise<void> {
```

Comment:
we should mark this method as deprecated while we're here (as we already have with `exportRoomKeys`)

Reply:
done